manifest.json

```diff
@@ -30,6 +30,14 @@
     ],
     "arguments": {
         "install" : [
+            {
+                "name": "disclaimer",
+                "type": "alert",
```

Comment:
Isn't it display_text ?

Reply:
display_text is the old stuff

Since version 4.3, alerts were introduced and correspond to colored message boxes. They are available in the config panel but also in install questions

Reply:
Oh, shouldn't you bump required version to yunohost 4.3 if using alerts?